src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public String getColor() {
+        return super.getColor();
+    }

```

Comment:
we don't need to override this method - you can just remove it from child figures
same everywhere